src/openrct2/core/File.cpp

```diff
@@ -120,11 +113,17 @@ namespace File
         return lines;
     }
 
+    void WriteAllBytes(const std::string& path, const void* buffer, size_t length)
```

Comment:
Why did you move this function? Makes the diff noisier than necessary.

Reply:
Because I find it more readable to have all Read* APIs in a row and then Write API.